File: Python/Cpu.py
from Komponen import Komponen

class Cpu(Komponen):
    __jumlahCore = 0
    __kecepatanGHz = 0.0

    def __init__(self, jumlahCore, kecepatanGHz, merk, nama):
        super().__init__(merk, nama)
        self.__jumlahCore = jumlahCore
        self.__kecepatanGHz = kecepatanGHz

    def get_jumlahCore(self):
        return self.__jumlahCore
    
    def get_kecepatanGHz(self):
        return self.__kecepatanGHz
    
    def set_jumlahCore(self, jumlahCore):
        self.__jumlahCore = jumlahCore

    def set_kecepatanGHz(self, kecepatanGHz):
        self.__kecepatanGHz = kecepatanGHz

File: Java/Cpu.java
public class Cpu extends Komponen
{
    private int jumlahCore;
    private double kecepatanGHz;

    public Cpu()
    {
        this.jumlahCore = 0;
        this.kecepatanGHz = 0;
    }

    public Cpu(int jumlahCore, double kecepatanGHz, String merk, String nama)
    {   
        super(merk, nama);
        this.jumlahCore = jumlahCore;
        this.kecepatanGHz = kecepatanGHz;
    }

    public void set_jumlahCore(int jumlahCore)
    {
        this.jumlahCore = jumlahCore;
    }

    public void set_kecepatanGHz(double kecepatanGHz)
    {
        this.kecepatanGHz = kecepatanGHz;
    }

    public int get_jumlahCore()
    {
        return this.jumlahCore;
    }

    public double get_kecepatanGHz()
    {
        return this.kecepatanGHz;
    }
}